src/scripts/influxdb-install.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+# License: https://github.com/influxdata/azure-resource-manager-influxdb-enterprise/blob/master/LICENSE
+#
+# Install IndluxDB OSS  ARM template cluster
+#
+# 06/2020 Initial Version
+#--------------------------
+
+help()
+{
+    echo " "
+    echo "This script configures a new InfluxDB OSS node for monitoring the enterprise cluster deployed with Azure ARM templates."
+    echo "Parameters:"
+    echo "-u  influxdb admin username"
+    echo "-p  influxdb admin password"
+    echo "-h  view this help content"
+}
+
+#########################
+# Logging func
+#########################
+
+# Custom logging with time so we can easily relate running times, also log to separate file so order is guaranteed.
+# The Script extension output the stdout/err buffer in intervals with duplicates.
+
+log()
+{
+
+     echo \[$(date +%d%m%Y-%H:%M:%S)\] "$1"
+     echo \[$(date +%d%m%Y-%H:%M:%S)\] "$1" >> /var/log/oss-install.log
+}
+
+log "Begin execution of InfluxDB OSS script extension on ${HOSTNAME}"
+START_TIME=$SECONDS
+
+#########################
+# Check user access
+#########################
+
+if [ "${UID}" -ne 0 ];
+then
+    log "Script executed without root permissions"
+    echo "You must be root to run this program." >&2
+    exit 3
+fi
+
+#Loop through options passed
+while getopts :u:p:h optname; do
+  log "Option $optname set"
+  case $optname in
+    u) #influxdb admin username
+      INFLUXDB_USER="${OPTARG}"
+      ;;
+    p) #influxdb admin password
+      INFLUXDB_PWD="${OPTARG}"
+      ;;
+    h) #show help
+      help
+      exit 2
+      ;;
+    \?) #unrecognized option - show help
+      echo -e \\n"Option -${BOLD}$OPTARG${NORM} not allowed."
+      help
+      exit 2
+      ;;
+  esac
+done
+
+#########################
+# Parameter handling
+#########################
+
+#Script Parameters
+OSS_VERSION="1.8.0"
+
+#########################
+# Installation 
+#########################
+
+install_influxdb()
```

Comment:
When we decide to update the monitor to InfluxDB v2, I think we should publish a new image for the monitor node with InfluxDB pre-installed. This is good for now.